api/src/main/java/io/opentelemetry/trace/SpanData.java

```diff
@@ -33,6 +33,9 @@
 /**
  * Immutable representation of all data collected by the {@link Span} class.
  *
+ * <p>{@link SpanData} is only for reporting out-of-bond {@link Span}s. Running {@link Span}s will
+ * be reported using SDK or proto.
```

Comment:
why not API? Running spans can be reported using pure API, correct?

Reply:
The only class for reporting in API is `SpanData`, which is not applicable to running `Span`s. Though `SpanSdk` has another way of reporting: 
https://github.com/open-telemetry/opentelemetry-java/blob/79ba36a00cae736055f5d1161205ab6115161895/sdk/src/main/java/io/opentelemetry/sdk/trace/SpanSdk.java#L39